### ISSUE TITLE
Adobe Digital Editions

### ISSUE THREAD

**Issue by whiskytango** [Winetricks/winetricks#991]
Please add ADE v2.0.1 to those on winetricks. You can get it from http://download.adobe.com/pub/adobe/digitaleditions/ADE_2.0_Installer.exe

**Comment by austin987** [comment]
Is there a reason you don't want to use 4.5? `adobe_diged4`

**Comment by austin987** [comment]
Looks like 2.0.1 may be broken under Wine:
https://bugs.winehq.org/show_bug.cgi?id=45235

**Comment by whiskytango** [comment]
Yes, there are problems. Do the comments at https://appdb.winehq.org/objectManager.php?sClass=version&iId=27978 help? Especially about installing  .NET 3.5 SP1? Thanks for following this up!

**Comment by whiskytango** [comment]
And there is the following recent complete how-to:
https://patdavid.net/2018/05/installing-adobe-digital-editions-on-linux-with-wine/

**Comment by bmaupin** [comment]
> Is there a reason you don't want to use 4.5? `adobe_diged4`

Among other things, ADE 2.0.1 is a prerequisite for a popular tool for removing DRM from ebooks.

**Comment by bmaupin** [comment]
I needed to install it recently, and this is what I used:

```
WINEARCH=win32 WINEPREFIX=~/.local/share/wineprefixes/adobe_diged2 winetricks -q dotnet35sp1
WINEPREFIX=~/.local/share/wineprefixes/adobe_diged2 winetricks -q windowscodecs
WINEPREFIX=~/.local/share/wineprefixes/adobe_diged2 winetricks -q corefonts
wget http://download.adobe.com/pub/adobe/digitaleditions/ADE_2.0_Installer.exe
WINEPREFIX=~/.local/share/wineprefixes/adobe_diged2 wine ADE_2.0_Installer.exe
```

```
$ wine --version
wine-5.0 (Ubuntu 5.0-3ubuntu1)
```

**Comment by austin987** [comment]
Looks like there was a regression. I opened https://bugs.winehq.org/show_bug.cgi?id=52815 for that (still in current wine, 7.6).
